lib/graph-assets.js

```diff
@@ -36,7 +36,7 @@ function node (state, createEdge) {
 
   tracker.on('change', function () {
     var list = tracker.list()
-    createEdge('list', Buffer.from(list))
+    createEdge('list', Buffer.from(list.join()))
```

Comment:
really minor but would you mind adding the ',' argument here? i know it's the default, but we explicitly pass ',' elsewhere too!